tests/regression/test_accounting.py

```diff
@@ -281,11 +270,6 @@ def test_accounting_cl_rebase_at_limits(accounting_oracle: Contract, lido: Contr
         eth_distributed_event["preCLBalance"] + rebase_amount == eth_distributed_event["postCLBalance"]
     ), "ETHDistributed: CL balance has not increased"
 
-    post_ttl_shares_event = _first_event(tx, PostTotalShares)
```

Comment:
because `LegacyOracle` has been disconnected from receiving updates, see: https://research.lido.fi/t/steth-on-optimism-upgrade-announcement-and-action-plan/8474#p-17840-legacyoracle-deprecation-notice-6